src/components/support-form/steps/ThankYou.tsx

```diff
@@ -17,6 +17,7 @@ const useStyles = makeStyles((theme: Theme) =>
     },
     instructions: {
       marginTop: theme.spacing(5),
+      marginLeft: theme.spacing(6),
```

Comment:
It might be better to define the margin equally on left and right by providing two arguments to the `theme.spacing(5, 6)` function

```suggestion
      margin: theme.spacing(5, 6),
```

I'm not really sure if it should be `5, 6` or `6, 5` though, please check